docs/project-setup/filesystem-sync.md

```diff
@@ -0,0 +1,73 @@
+# Filesystem Sync with Unison
+
+The default NFS filesystems that Outrigger provides are very easy to setup and easy to share code and other files into
+your project containers, but NFS can be slow when writing, reading, or scanning thousands of files when compared to
+native filesystem performance. This can be a particular challenge when building your code base from a package manager onto
+a volume mounted NFS share.
+
+It also has the downside of not propagating filesystem modification events, which can be problematic if you wanted to have
```

Comment:
NFS also has the downside . . . . . . are triggered due to local editing via your IDE.